File: library/src/main/java/name/zeno/android/util/MD5.java
package name.zeno.android.util;

@SuppressWarnings("unused")
public class MD5
{
  /**
   * @deprecated use {@link Encode#md5(String)}
   */
  public static String md5(String string)
  {
    return Encode.md5(string);
  }

  /**
   * @deprecated use {@link Encode#md5(byte[])}
   */
  public static String md5(byte[] buffer)
  {
    return Encode.md5(buffer);
  }
}

File: library/src/main/java/name/zeno/android/util/MD5.kt
package name.zeno.android.util

object MD5 {

  @Deprecated("use Encode#md5(String)")
  fun md5(string: String): String? {
    return Encode.md5(string)
  }


  @Deprecated("use Encode#md5(byte[])")
  fun md5(buffer: ByteArray): String? {
    return Encode.md5(buffer)
  }
}
